src/main/java/com/conveyal/gtfs/stats/model/StopStatistic.java

```diff
@@ -0,0 +1,25 @@
+package com.conveyal.gtfs.stats.model;
+
+import com.conveyal.gtfs.stats.StopStats;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+/**
+ * Created by landon on 10/4/16.
+ */
+public class StopStatistic {
+    public String stop_id;
```

Comment:
Do we need a separate class or can we just use stopstats and mark things transient/@JsonIgnore?